### DISCUSSION TITLE
ssh on rAudio-1 latest version

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#112] [General]
After testing Volumio, Moode, I made a test of the latest version of rAudio-1. 
I've to say that it is exactly what I'm waiting for : A very good Ui and a very good sound quality. Really better than Volumio and very closed to Moode (maybe better !).
Congrats to the developers. 
Now I'm facing to SSH problem : 
I can log using root@the pi IP address
but the password : rune seems not working as I get the message Access denied...
any idea ?
I've to install a script to use the audio phonics spc pi V2  with a led lighting such button. 
Thanks for your help !

**Comment by rern** [comment]
It's `ros`.

**Reply by Rasb74** [reply]
Many thanks ! I spent many hours to look at this !!!!!!
